integration/query_compat_test.go

```diff
@@ -32,7 +32,7 @@ type queryCompatTestCase struct {
 	sort           bson.D                   // defaults to `bson.D{{"_id", 1}}`
 	projection     bson.D                   // nil for leaving projection unset
 	resultType     compatTestCaseResultType // defaults to nonEmptyResult
-	resultPushdown bool                     // TODO https://github.com/FerretDB/FerretDB/issues/1279
+	resultPushdown bool                     // deafault to false
```

Comment:
Ah, now I got it! So, it's false by default, and in the test cases where pushdown is used, you added true!

Reply:
Yeah I believe it's good for now, although I can see that if we start pushdown more generic filters, it will need to be set for too much test cases. But we can make this check more complex in any time :)